docs/tables/aws_organizations_organizational_unit.md

```diff
@@ -0,0 +1,21 @@
+# Table: aws_organizations_organizational_unit
+
+A container for accounts within a root. An OU also can contain other OUs, enabling you to create a hierarchy that resembles an upside-down tree, with a root at the top and branches of OUs that reach down, ending in accounts that are the leaves of the tree. When you attach a policy to one of the nodes in the hierarchy, it flows down and affects all the branches (OUs) and leaves (accounts) beneath it. An OU can have exactly one parent, and currently each account can be a member of exactly one OU.
+
+**Note**: The `parent_id` is the required to make the API call. It is the unique identifier (ID) of the root or OU whose child OUs you want to list.
```

Comment:
Your second example in this ticket *does not* specify `parent_id`, how come it worked?

Reply:
> Your second example in this ticket _does not_ specify `parent_id`, how come it worked?

In the table definition we have two config one is [List Config](https://github.com/turbot/steampipe-plugin-aws/pull/1677/files#diff-a5533f317b4a9cd8f0738573bf490499b95488f94416241aad2bcd8729fcd650R27) and another one is [Get Config](https://github.com/turbot/steampipe-plugin-aws/pull/1677/files#diff-a5533f317b4a9cd8f0738573bf490499b95488f94416241aad2bcd8729fcd650R20).
In List config we generally make list API call and in Get config we make the get API call.

Here we are using [ListAccountsForParent](https://docs.aws.amazon.com/organizations/latest/APIReference/API_ListOrganizationalUnitsForParent.html) API in the `List config`, for making the `ListAccountsForParent` API call we must need to pass the `ParentId` in param of this API call, so `parent_id` is required to make list API call.

In Get config we are using the [DescribeOrganizationalUnit](https://docs.aws.amazon.com/organizations/latest/APIReference/API_DescribeOrganizationalUnit.html) API call, for making this API call we must have to pass the `OrganizationalUnitId`, we do not need `ParentId` for this API call.

The Get API will be called if we are providing `id` as query parameter, so in second example we need to pass the `parent_id`.

Reply:
Got it for the logic underneath ; may I suggest a working like this? 

```suggestion
You **_must_** specify a single `parent_id` or `id` in a where or join clause in order to use this table.
```
I found similar wording/format for [aws_route53_record](https://hub.steampipe.io/plugins/turbot/aws/tables/aws_route53_record), [aws_iam_access_advisor](https://hub.steampipe.io/plugins/turbot/aws/tables/aws_iam_access_advisor), [aws_cloudtrail_trail_event](https://hub.steampipe.io/plugins/turbot/aws/tables/aws_cloudtrail_trail_event) and others

Also from a user perspective I find it odd that we can't have the listing without a parent_id, but looking at API reference it looks tricky to get the root id reliably...

Reply:
Yeah @ajoga, this looks better, will push the changes. Thanks!